src/pages/index.mdx

```diff
@@ -12,35 +8,134 @@ The developer docs will teach you how to build applications, issue and use token
+| **Data** | [Horizon introduction](https://developers.stellar.org/docs/data/horizon) | Access network data using the Horizon REST API |
+
+| **Decentralized applications** | _Start by viewing the smart contract wayfinding section_ |  |
+| :-: | --- | --- |
+| **Learn** | [Transaction simulation](https://developers.stellar.org/docs/learn/encyclopedia/contract-development/contract-interactions/transaction-simulation) | A process that allows developers to test the outcome of a transaction without actually executing it, helping to ensure accuracy and prevent errors |
+| **Guide** | [Dapp frontend](https://developers.stellar.org/docs/build/apps/dapp-frontend) | Create a web app that interacts with the contracts via RPC calls |
+| **Guides** | [Dapp how-to guides](https://developers.stellar.org/docs/build/guides/dapps) | Explore the various guides relating to dapp development |
+| **Tool** | [Stellar Wallets Kit](https://stellarwalletskit.dev/) | A kit to handle all Stellar Wallets at once with a simple API and without caring about individual configurations for each one of them |
+| **Data** | [Stellar JS SDK](https://stellar.github.io/js-stellar-sdk/) | Communicate with a Horizon server and Soroban RPC |
+| **Data** | [RPC introduction](https://developers.stellar.org/docs/data/rpc) | Access network data using the RPC |
+
+| **Exchanges** |  |  |
+| :-: | --- | --- |
+| **Tools** | [SDKs](https://developers.stellar.org/docs/tools/sdks) | Used to connect to Horizon or RPC, as well as construct and parse transactions |
+| **Data** | [Run your own RPC](https://developers.stellar.org/docs/data/rpc/admin-guide) | Exchanges typically run their own RPC or Horizon to maintain greater control over data integrity, performance, and security |
+| **Data** | [Run your own Horizon](https://developers.stellar.org/docs/data/horizon/admin-guide) | Exchanges typically run their own RPC or Horizon to maintain greater control over data integrity, performance, and security |
+
+### Stellar for infrastructure providers
+
+| **Horizon** |  |  |
+| :-: | --- | --- |
+| **Learn** | [Hardware requirements](https://developers.stellar.org/docs/data/requirements) | The recommended hardware requirements for users looking to set up a Horizon instance |
+| **Tutorial** | [Admin guide](https://developers.stellar.org/docs/data/requirements) | A comprehensive set of guides that will teach you how to administer a production Horizon instance |
+| **Guide** | [API reference](https://developers.stellar.org/docs/data/horizon/api-reference) | Detailed documentation that provides information about the API endpoints, methods, parameters, and responses |
+| **Tools** | [SDK library](https://developers.stellar.org/docs/tools/sdks/library) | Use the various SDKs to simplify the Horizon setup process by using pre-built tools and libraries that make it easier to configure, manage, and connect to the network |
+
+| **RPC** |  |  |
```

Comment:
For Horizon and RPC is it too meta to link to the `Ecosystem Providers` for the infra provider persona?
https://developers.stellar.org/docs/data/rpc/rpc-providers
https://developers.stellar.org/docs/data/horizon/horizon-providers

Reply:
I think we _could_ list the Ecosystem Providers in the context of like, I've set up a Horizon/RPC and now I want to list it in this list. Thoughts?

Reply:
Yeah I like that